scripts/capture-scrutinize.sh

```diff
@@ -49,34 +60,106 @@ do
     esac
 done
 
+logInfo "All output saved to $HOST_OP_DIR"
 mkdir -p $HOST_OP_DIR
+
+logInfo "Save off different k8s objects"
+for obj in verticadbs pods statefulsets deployments
+do
+    kubectl get $obj -A -o yaml > $HOST_OP_DIR/$obj.yaml
+done
+
+logInfo "Save off the OLM operator log if deployed"
+if kubectl get ns olm 2> /dev/null
+then
+    kubectl logs -n olm -l app=olm-operator --tail=-1 > $HOST_OP_DIR/olm-operator.log
+fi
+
+logInfo "Collect scrutinize for VerticaDB clusters"
 for ns in $NS
```

Comment:
It is not too important but is it possible to break some parts into functions to make it more readable?